package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@nmshd/connector",
-    "version": "3.7.1",
+    "version": "4.1.0",
```

Comment:
I don't see any breaking changes, `4.8.0` should be safe.

Reply:
i anticipated that https://github.com/nmshd/connector/pull/134 gets merged first and there for already changed to 4.1.0

Reply:
ah, i see. But 134 was no breaking change.

Reply:
Yea just saw that too, I misread the SDK version